apps/guardian-ui/src/components/GuardiansCard.tsx

```diff
@@ -0,0 +1,95 @@
+import { Card, CardBody, CardHeader, Text } from '@chakra-ui/react';
+import { StatusIndicator, Table, TableColumn, TableRow } from '@fedimint/ui';
+import { useTranslation } from '@fedimint/utils';
+import React, { useMemo } from 'react';
+import { ConfigResponse, PeerConnectionStatus, StatusResponse } from '../types';
+
+type TableKey = 'name' | 'status' | 'health' | 'lastContribution';
+
+interface Props {
+  status: StatusResponse | undefined;
+  config: ConfigResponse | undefined;
+}
+
+export const GuardiansCard: React.FC<Props> = ({ status, config }) => {
+  const { t } = useTranslation();
+
+  const columns: TableColumn<TableKey>[] = useMemo(
+    () => [
+      {
+        key: 'name',
+        heading: t('federation-dashboard.guardians.name-label'),
+      },
+      {
+        key: 'status',
+        heading: t('federation-dashboard.guardians.name-label'),
+      },
+      {
+        key: 'health',
+        heading: t('federation-dashboard.guardians.health-label'),
+      },
+      {
+        key: 'lastContribution',
+        heading: t('federation-dashboard.guardians.last-contribution-label'),
+      },
+    ],
+    [t]
+  );
+
+  const rows: TableRow<TableKey>[] = useMemo(() => {
```

Comment:
We should still surface "Peer ID" as a value to guardians since this will be how their fedimints are referenced in logs. If I'm a Guardians troubleshooting logs, I don't want the mental burden of remembering if Alice or Bob were Peer 0 or Peer 1.

Reply:
Agreed, I'll add another column and a note about deviating from designs, similar to how I did with `Gateway ID`

Reply:
Design is updating Figma (see Discord)